pom.xml

```diff
@@ -245,6 +245,11 @@
             <artifactId>jbcrypt</artifactId>
             <version>0.3m</version>
         </dependency>
+        <dependency>
+            <groupId>org.flywaydb</groupId>
+            <artifactId>flyway-core</artifactId>
+            <version>5.2.1</version>
```

Comment:
Could you please opt for adding a version property here? Also, v5.2.3 has been released in the meantime.